src/Microsoft.OData.Core/ODataMessageWriter.cs

```diff
@@ -306,14 +306,14 @@ public Task<ODataWriter> CreateODataDeltaResourceSetWriterAsync(IEdmEntitySetBas
         /// Asynchronously creates an <see cref="ODataWriter" /> to write a delta resource set.
         /// </summary>
         /// <param name="entitySet">The entity set we are going to write entities for.</param>
-        /// <param name="entityType">The entity type for the entries in the resource set to be written (or null if the entity set base type should be used).</param>
+        /// <param name="structuredType">The structured type for the entries in the resource set to be written (or null if the entity set base type should be used).</param>

```

Comment:
>entity [](start = 127, length = 6)

also change to "resource set"

Reply:
actually, I reverted this change. The delta resource set always contains entity types, so it's correct to say entity set and entitytype.. Should probably also change CreateODataDeltaResourceSetWriter on the context in 8.0, but it would be breaking to do so in a 7.x release.

---
In reply to: [427634277](https://github.com/OData/odata.net/pull/1770#discussion_r427634277) [](ancestors = 427634277)